packages/react-components/src/atoms/Tag.tsx

```diff
@@ -10,29 +10,23 @@ const containerStyles = css({
   justifyContent: 'center',
   alignItems: 'center',
 
-  marginTop: '6px',
-  marginBottom: '6px',
-
-  [`@media (min-width: ${tabletScreen.width}px)`]: {
```

Comment:
Are we sure we want to remove this?

Reply:
I checked the design guide and it just says 12 top 6 bottom, no conditions

Reply:
![image](https://user-images.githubusercontent.com/763528/90408110-4decae00-e09f-11ea-9e2b-49cf1380ad01.png)

It's based on this.

Reply:
Yeah, it matches that now, expect I made the padding (see Measure>Right side) 7 top 5 bottom because James asked me to move the text down by 1px in Design QA